docs/requirements.txt

```
@@ -1,4 +1,4 @@
-Sphinx==3.1.2
+sphinx==3.5.3
```

Is it required to write the version name? It would just give the new version every time a new one releases and we won't have a problem changing it.
```suggestion
sphinx
```

---

sounds good as well!